fe/fe-core/src/main/java/org/apache/doris/nereids/trees/expressions/Divide.java

```diff
@@ -62,14 +63,15 @@ public DataType getDataType() throws UnboundException {
     @Override
     public DecimalV3Type getDataTypeForDecimalV3(DecimalV3Type t1, DecimalV3Type t2) {
         int retPercision = t1.getPrecision() + t2.getScale();
-        int retScale = t1.getScale();
+        int retScale = t1.getScale() + t2.getScale()
+                + Config.div_precision_increment;
```

Comment:
should be the first argument scale + div_precision_increment ? 
```suggestion
        int retScale = t1.getScale() + Config.div_precision_increment;
```